src/main/java/frc/robot/subsystems/conveyor/ConveyorConstants.java

```diff
@@ -1,11 +1,26 @@
 package frc.robot.subsystems.conveyor;
 
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
 import edu.wpi.first.units.*;
+import edu.wpi.first.wpilibj.Timer;
+import lib.webconstants.LoggedTunableNumber;
 
 public class ConveyorConstants {
 
     public static final MutableMeasure<Velocity<Angle>> FEED_VELOCITY =
             Units.RotationsPerSecond.of(10).mutableCopy();
     public static final MutableMeasure<Dimensionless> SETPOINT_TOLERANCE =
             Units.Value.of(0.05).mutableCopy();
+    public static final double GEAR_RATIO = 0;
+    public static LoggedTunableNumber KP = new LoggedTunableNumber("Conveyor/kP", 0);
+    public static LoggedTunableNumber KI = new LoggedTunableNumber("Conveyor/kI", 0);
+    public static LoggedTunableNumber KD = new LoggedTunableNumber("Conveyor/kD", 0);
+
+    public static LoggedTunableNumber KS = new LoggedTunableNumber("Conveyor/kS", 0);
+    public static LoggedTunableNumber KV = new LoggedTunableNumber("Conveyor/kV", 0);
+    public static LoggedTunableNumber KA = new LoggedTunableNumber("Conveyor/kA", 0);
```

Comment:
If you give the tunable number a value, that will be the default value no matter what you do, which means you'll have pid zero. You have to use the constructor that only gets the name of the constant.